lredis/commands.lua

```diff
@@ -40,6 +41,48 @@ function methods:client_pause(delay)
 	return handle_ok_or_err(resp)
 end
 
+function methods:hmget(key, ...)
+	local resp = self:call("HMGET", key, ...)
+	if type(resp) == "table" then
+		local ret = {}
+		for i, v in pairs({...}) do
```

Comment:
I see your point... but there's no `table.pack` in lua 5.1, so i think for consistency i'd rather just use `ipairs()`